### PR TITLE
Update pages for GSoC 2018

### PR DESCRIPTION
**Pull request by marc1706**

### CODE REVIEW

File: src/AppBundle/Resources/views/Development/gsoc.html.twig [hunk 1]
```diff
@@ -63,10 +63,44 @@
...
+	<h4 class="gsoc-page">Community Bonding Period</h4>
+	<p>
+		The community bonding period is the first phase after you're accepted as a student in the Google Summer of Code. Hang around IRC and the forums
+		to get to know better the phpBB community, the phpBB team members, and your mentors for Google Summer of Code. This initial phase helps with
+		getting to know each other and growing more accustomed to the communication via forum posts and IRC. It can and should also be used to
+		<ul>
+			<li>get feedback on your proposal</li>
+			<li>further discuss and clarify your ideas for the project</li>
+			<li>set up your development environment</li>
+			<li>prepare yourself and your project plan for the months that lie ahead of your, e.g. prepare a clear schedule, milestones, etc.</li>
```
Comment (hanakin) [inline src/AppBundle/Resources/views/Development/gsoc.html.twig]: ahead of your -> ahead of you

File: src/AppBundle/Resources/views/Development/gsoc.html.twig [hunk 2]
```diff
@@ -63,10 +63,44 @@
...
+		</ul>
+	</p>
+	<h4 class="gsoc-page">Coding Period</h4>
+	<p>
+		Once you have layed out a clear schedule during the Community Bonding Period, communication between you and your mentor is vital to the
```
Comment (hanakin) [inline src/AppBundle/Resources/views/Development/gsoc.html.twig]: layed -> laid

**Comment by paul999** [comment]
My main point is that I don't want to have a student for the website, mostly because it is not our real product (And that is where GSOC is for, and it is frowned upon by doing this types of projects). Also, looking at the amount of applications we have for the website in the past, I don't think it is actually worth in having one.

**Comment by hanakin** [comment]
I would agree if they want something front-end for a project have them redo all the js! 🤮😜

**Comment by marc1706** [comment]
Ok, will remove the website part then (just left it in there because it was in there anways :smile: )

**Comment by marc1706** [comment]
Should add stricter requirements for students, i.e. make a pull request with a minimum of 10 changed lines mandatory.

**Pull request merged**